iptables/tables.sls

```diff
@@ -4,15 +4,21 @@
 {%- from "iptables/map.jinja" import firewall with context %}
 
 {%- for t in ['filter','nat','mangle'] %}
-  {%- for cn, cv in firewall.get(t)|dictsort %}
+  {%- for cn, cv in firewall.get(t).items() %}
     {%- set pol = cv.policy | default('ACCEPT') %}
     {%- set rules = cv.rules | default({}) %}
 
 chain_present_{{ t }}_{{ cn }}:
   iptables.chain_present:
     - table: {{ t }}
     - name: {{ cn }}
+  {%- endfor %}
+{%- endfor %}
 
+{%- for t in ['filter','nat','mangle'] %}
```

Comment:
Perhaps the same can be done here. Removing [these lines](https://github.com/saltstack-formulas/iptables-formula/pull/48/files#diff-900a92f9dfe3ca867e6697161bb736baR16-R18) we can get a single outer loop with two repeated inner loops, one to create the chains and the other to create the rules, like

```jinja
{%- for t in ['filter','nat','mangle'] %}
  {%- for cn, cv in firewall.get(t).items() %}
  ...
  {# chains creation#}
  ...
  {% endfor %}

  {%- for cn, cv in firewall.get(t).items() %}
    {%- set pol = cv.policy | default('ACCEPT') %}
    {%- set rules = cv.rules | default({}) %}
    {%- for rn, rv in rules.items() %}
    ...
    {# rules creation#}
    ...
  {% endfor %}
{% endfor %}
```

Reply:
Seems more elegant yes